airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/ResetActivity.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling.activities;
+
+import io.airbyte.config.StreamDescriptor;
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+import java.util.List;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@ActivityInterface
+public interface ResetActivity {
```

Comment:
nit: maybe this should be called StreamResetActivity? I don't feel super strongly here though